### PR TITLE
jmeter: add page

### PR DESCRIPTION
**Pull request by andre-santos-fd**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: jmeter.md
Comment (navarroaxel) [inline jmeter.md:6]: Hi @andre-santos-fd, welcome and thanks for your submission. All these examples are in CLI mode?

**Comment by CleanMachine1** [comment]
Hey, could you please move this file into a correct directory. pages/common/jmeter.md

**Comment by kbdharun** [comment]
> Hey, could you please move this file into a correct directory. pages/common/jmeter.md

Yeah, I too think the same, Since it's cross platform we can place it in `pages/common`

File: pages/common/jmeter.md
Comment (kbdharun) [inline pages/common/jmeter.md:3]:
```suggestion
> Open source java application designed for load testing functional behavior and measure performance.
```

File: pages/common/jmeter.md
Comment (kbdharun) [inline pages/common/jmeter.md:4]:
```suggestion
> More information: <https://jmeter.apache.org>.
```
I don't think the last slash is necessary.

**Review by kbdharun — CHANGES_REQUESTED** [review]

File: jmeter.md
Comment (andre-santos-fd) [reply jmeter.md]: Hi @navarroaxel, thank you! Yes all the examples are in CLI because of the `-n` flag.

File: pages/common/jmeter.md
Comment (andre-santos-fd) [reply pages/common/jmeter.md]: Done, please validate! ✅

File: pages/common/jmeter.md
Comment (andre-santos-fd) [reply pages/common/jmeter.md]: Done, please validate! ✅

File: pages/common/jmeter.md
Comment (CleanMachine1) [inline pages/common/jmeter.md:9]:
```suggestion

```

File: pages/common/jmeter.md
Comment (EmilySeville7cfg) [inline pages/common/jmeter.md:6]:
```suggestion
- Run in [n]ongui mode for a specific [t]est plan:
```

File: pages/common/jmeter.md
Comment (EmilySeville7cfg) [inline pages/common/jmeter.md:8]:
```suggestion
`jmeter --nongui --testfile {{path/to/file}}.jmx`
```

```suggestion
`jmeter --nongui --testfile {{path/to/file.jmx}}`
```

Is `.jmx` a required extension? Or it can be any? Choose the second suggestion if extension is required but it doesn't matter what it is.

File: pages/common/jmeter.md
Comment (EmilySeville7cfg) [inline pages/common/jmeter.md:12]: - We prefer long options. There is no fixed convention in our style guide when we do this. But it will be added soon.
- Don't use plain `name` or `logfile` as above. They are paths here so they must be written as:

```suggestion
`jmeter --nogui --testfile {{path/to/file}}.jmx --logfile {{path/to/logfile}}.jtl`
```

```suggestion
`jmeter --nogui --testfile {{path/to/file.jmx}} -l {{path/to/logfile.jtl}}`
```

File: pages/common/jmeter.md
Comment (EmilySeville7cfg) [inline pages/common/jmeter.md:10]:
```suggestion
- Specify a specific log file:
```

File: pages/common/jmeter.md
Comment (EmilySeville7cfg) [inline pages/common/jmeter.md:14]:
```suggestion
- Use a specific proxy server and a specific port:
```

File: pages/common/jmeter.md
Comment (EmilySeville7cfg) [inline pages/common/jmeter.md:18]:
```suggestion
- Define user variables:
```

File: pages/common/jmeter.md
Comment (EmilySeville7cfg) [inline pages/common/jmeter.md:16]: It's better to provide a sample value instead of `proxy_server_url`:

```suggestion
`jmeter --nongui --testfile {{path/to/file}}.jmx --proxyHost {{proxy_server_url}} --proxyPort {{port}}`
```

```suggestion
`jmeter --nongui --testfile {{path/to/file.jmx}} --proxyHost {{proxy_server_url}} --proxyPort {{port}}`
```

File: pages/common/jmeter.md
Comment (EmilySeville7cfg) [inline pages/common/jmeter.md:20]:
```suggestion
`jmeter --jmeterproperty {{key}}='{{value}}' --nongui --testfile {{path/to/file}}.jmx`
```

```suggestion
`jmeter --jmeterproperty {{key}}='{{value}}' --nongui --testfile {{path/to/file.jmx}}`
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/common/jmeter.md
Comment (andre-santos-fd) [reply pages/common/jmeter.md]: Makes sense, please validate ✅

File: pages/common/jmeter.md
Comment (andre-santos-fd) [reply pages/common/jmeter.md]: It is a required extension for jmeter to boot up in this case. 
Please validate! ✅

File: pages/common/jmeter.md
Comment (andre-santos-fd) [reply pages/common/jmeter.md]: Yep, looks more readable. 
Please validate ✅

File: pages/common/jmeter.md
Comment (andre-santos-fd) [reply pages/common/jmeter.md]: ✅

File: pages/common/jmeter.md
Comment (andre-santos-fd) [reply pages/common/jmeter.md]: ✅

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages/common/jmeter.md
Comment (andre-santos-fd) [reply pages/common/jmeter.md]: ✅

File: pages/common/jmeter.md
Comment (andre-santos-fd) [reply pages/common/jmeter.md]: I was not too sure if I should specify or not.
Please validate ✅

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages/common/jmeter.md
Comment (EmilySeville7cfg) [inline pages/common/jmeter.md:9]:
```suggestion
`jmeter --nongui --testfile {{path/to/file}}.jmx`

- Use a specific log file:
```

File: pages/common/jmeter.md
Comment (kbdharun) [inline pages/common/jmeter.md:3]: I have a suggestion should this line be 
```suggestion
> Open source java application designed for load testing functional behavior and measuring performance.
```
 or 
```suggestion
> Open source java application designed for load testing functional behavior and to measure performance.
 ```
 I think the current one feels a bit odd with *measure perforamance* alone. What do you guys think @navarroaxel @CleanMachine1 @EmilySeville7cfg ?

File: pages/common/jmeter.md
Comment (navarroaxel) [inline pages/common/jmeter.md:6]: Perhaps this better
```suggestion
- Run a specific test plan in nongui mode:
```

File: pages/common/jmeter.md
Comment (navarroaxel) [inline pages/common/jmeter.md:10]: Isolated examples are best, and self explanatory
```suggestion
- Run a test plan in nongui mode using a specific log file:
```

File: pages/common/jmeter.md
Comment (navarroaxel) [reply pages/common/jmeter.md:3]: I prefer leave as-is.

File: pages/common/jmeter.md
Comment (navarroaxel) [inline pages/common/jmeter.md:14]: I think port and host are implicit in "proxy" 
```suggestion
- Run a test plan in nongui mode using a specific proxy:
```

File: pages/common/jmeter.md
Comment (navarroaxel) [inline pages/common/jmeter.md:18]: The man page <https://manned.org/jmeter> refers to this variables as `JMeter properties` so:
```suggestion
- Run a test plan in nongui mode using a specific JMeter property:
```

**Review by kbdharun — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Merged, thanks! 🚀

**Comment by navarroaxel** [comment]
@EmilySeville7cfg @kbdharun Please when merge only left the commit subject untouched, remove the rest commit message's body like the following:
```
Co-authored-by: ...
```
The PR's authors should have all the credit for their work.

@EmilySeville7cfg I asked for a new review from @CleanMachine1, please wait a little before merge.

**Comment by EmilySeville7cfg** [comment]
What about to restrict merging when some of reviewers are not reviewed changes in PRs?

**Comment by navarroaxel** [comment]
maybe I should start using `request changes` more often to block the merge and take my time to re-review. I stop using it because IMO `J Doe requested changes` sounds more imperative for contributors instead of `left comments`.

**Comment by EmilySeville7cfg** [comment]
#8395 is opened. Please leave comments there. :)

**Pull request merged**
